flux/clusters/k8s-01/user-profile-api-db-exp/overlays/DEV/postgrescluster-replication.yaml

```
@@ -0,0 +1,5 @@
+# Always replace placeholder instances from `base`
+- op: add
+  path: /spec/patroni/dynamicConfiguration/postgresql/create_replica_methods
+  value: 
```

yamllint

[error] trailing spaces

(trailing-spaces)